JavascriptMornings/Maitee
# Create html

## Review checklist


## Testing

Steps to test the change:
1. Pull the Branch
2. Open index.html file in navigator
3. check if buttons `Plug/Unplug seat belt` and `Open/Close Trunk` are working. 
## Additional notes
![image](https://user-images.githubusercontent.com/29434543/35643257-211101b8-069c-11e8-9d1a-3fab268cece3.png)


## Deploy checklist
